srcd/superset/assets/stylesheets/less/custom-brand.less

```diff
@@ -18,11 +18,213 @@
+
+@brand-primary:         #00A699;
+@brand-success:         #4AC15F;
+@brand-info:            lighten(#2AB7CA, 15%);
+@brand-warning:         #FED766;
+@brand-danger:          #FE4A49;
+
+//== Scaffolding
+//
+//## Settings for some of the most global styles.
+
+//** Background color for `<body>`.
+@body-bg:               #f5f5f5;
+//** Global text color on `<body>`.
+@text-color:            @gray-dark;
+
+//** Global textual link color.
+@link-color:            @brand-primary;
+//** Link hover color set via `darken()` function.
+@link-hover-color:      darken(@link-color, 15%);
+//** Link hover decoration.
+@link-hover-decoration: underline;
+
+//== Soft-Branding 
+ 
 @brand-primary:         #00b491;
 @brand-secondary:       #8719cb;
 @brand-tertiary:       	#f89c30;
 
 
+//== Typography
+//
+//## Font, line-height, and color for body text, headings, and more.
+
+@font-face {
+  font-family: 'Source Sans Pro', 'Helvetica Neue', Arial, sans-serif;
+  src: url('../fonts/SourceSansPro-Regular.ttf')  format('truetype');
+}
+
+@font-face {
+  font-family: 'Source Code Pro', 'Helvetica Neue', Arial, sans-serif;
+  src: url('../fonts/SourceCodePro-Regular.ttf')  format('truetype');
+}
+
+@font-family-sans-serif:  'Source Sans Pro', 'Helvetica Neue', Arial, sans-serif;
+
+@font-family-serif:       Georgia, "Times New Roman", Times, serif;
+//** Default monospace fonts for `<code>`, `<kbd>`, and `<pre>`.
+@font-family-monospace:   Menlo, Monaco, Consolas, "Courier New", monospace;
+@font-family-base:        @font-family-sans-serif;
+
+@font-size-base:          16px;
+@font-size-large:         ceil((@font-size-base * 1.25)); // ~18px
+@font-size-small:         ceil((@font-size-base * 0.85)); // ~12px
+
+@font-size-h1:            floor((@font-size-base * 2.6)); // ~36px
+@font-size-h2:            floor((@font-size-base * 2.15)); // ~30px
+@font-size-h3:            ceil((@font-size-base * 1.7)); // ~24px
+@font-size-h4:            ceil((@font-size-base * 1.25)); // ~18px
+@font-size-h5:            @font-size-base;
+@font-size-h6:            ceil((@font-size-base * 0.85)); // ~12px
+
+//** Unit-less `line-height` for use in components like buttons.
+@line-height-base:        1.428571429; // 20/14
+//** Computed "line-height" (`font-size` * `line-height`) for use with `margin`, `padding`, etc.
+@line-height-computed:    floor((@font-size-base * @line-height-base)); // ~20px
+
+//** By default, this inherits from the `<body>`.
+@headings-font-family:    @font-family-base;
+@headings-font-weight:    300;
+@headings-line-height:    1.1;
+@headings-color:          inherit;
+
+
+//== Typography overwrite
+
+body {
+	font-family: 'Source Sans Pro', 'Helvetica Neue', Arial, sans-serif;
+	font-size: 16px;
+}
+
+.table-condensed {
+	font-size: 14px !important;
+}
+
+h2 {
+	font-family: 'Source Sans Pro', 'Helvetica Neue', Arial, sans-serif;
+	font-size: @font-size-h2;
+	font-weight: 600; 
+}
+
+h2:after {
+	content: "";
+    position: absolute;
```

Comment:
please use some proper text-editor that can re-format your code automatically.
For example, I use VS Code and Prettier extension. It will make your code consistent every time you save file.

Reply:
Thank you, will do so.